src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkflowComponent.scala

```diff
@@ -86,6 +88,15 @@ trait WorkflowComponent {
     def workflowFailure = foreignKey("FK_WF_ERR_FAILURE", workflowFailureId, workflowFailureQuery)(_.id)
   }
 
+  class WorkflowAuditStatusTable(tag: Tag) extends Table[WorkflowAuditStatusRecord](tag, "AUDIT_WORKFLOW_STATUS") {
```

Comment:
Could you add a comment explaining the purpose of this table?  It's not immediately clear to me.  Is it recording the times of every status transition for every workflow?